definitions/Crane/CraneIsPlaying.json

```diff
@@ -0,0 +1,23 @@
+{
+	"$schema": "../../schemas/event-schema.json",
+	"$meta": {
+		"name": "GraneIsPlaying",
```

Comment:
I assume this should say `CraneIsPlaying` and not `GraneIsPlaying`? I say this so you can also check your code where it's sent.